### PR TITLE
feat: integrate beehiiv newsletter subscription

### LINKED ISSUE #1209: Migrate newsletter subscription widget to use beehiv

### PR DESCRIPTION
**Pull request by andyesp**
Closes #1209

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -92,8 +92,12 @@ DISCORD_CHANNEL_ID=
...
 
-#DCL Data
+# DCL Data
 GATSBY_DCL_DATA_API=https://data.decentraland.vote/
 
 # Rollbar Token
 DAO_ROLLBAR_TOKEN=""
+
+# Newsletter
+BEEHIIV_API_KEY=
```
Comment (1emu) [inline .env.example:102]: we need this in definitions as a secret right?

**Review by 1emu — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (andyesp) [reply .env.example:102]: Yessss

**Review by ncomerci — APPROVED** [review]

File: .env.example [hunk 2]
```diff
@@ -92,8 +92,12 @@ DISCORD_CHANNEL_ID=
...
+BEEHIIV_PUBLICATION_ID=
```
Comment (ginocingolani) [inline .env.example:103]: We need to update the PUBLICATION_ID

Comment (ginocingolani) [inline .env.example:103]: We need to update this ID.

**Review by ginocingolani — CHANGES_REQUESTED** [review]

File: .env.example (hunk 2 above)
Comment (andyesp) [reply .env.example:103]: Updated

**Review by ginocingolani — APPROVED** [review]
LGTM

**Pull request merged**